pyproject.toml

```diff
@@ -19,6 +19,7 @@ cohere = "^4.32"
 numpy = "^1.25.2"
 pinecone-text = "^0.7.0"
 colorlog = "^6.8.0"
+scikit-learn = "^1.3.2"
```

Comment:
`scikit-learn` is a very heavy library — I'd rather we avoid this at all costs, should be possible to implement TFIDF with pure numpy

Reply:
Thanks James, I'll get that changed.

Reply:
<img width="713" alt="image" src="https://github.com/aurelio-labs/semantic-router/assets/20096648/6981da24-24db-47a8-897a-b884bc232e84">